### PR TITLE
[4.0] Make the IFW styling compile with SCSS, nuke most unused rules

### PR DESCRIPTION
**Pull request by mbabker**
### Summary of Changes

This will restructure the CSS for the install from web plugin's media so that it is compiled through a SCSS file.  This is going to have a couple of benefits:

- Able to use the Bootstrap mixin for the badge style variants (the server markup should be updated to go back to using these custom variants versus the generic success/warning/whatnot)
- Able to use the Bootstrap gradient mixin
- Gets the same post processing as the templates (i.e. browser support)

Since I built the layouts for 4.0 support only using the Bootstrap 4 structures and not really implementing anything custom, most of the style rules for the 3.x layouts are removed and at this point the only style rules left are things actively in use or things that need to be re-implemented (i.e. the `.fader` which creates the gradient effect at the bottom of cards so there isn't a hard break where the text cuts off).

### Testing Instructions

With PR applied, validate the IFW tab renders the same as before.

### CODE REVIEW

File: media/plg_installer_webinstaller/scss/client.scss [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    // Bootstrap 4 does not indent second level navigation
+    .nav {
+      .nav {
+        padding-left: 15px;
+      }
+    }
+
+    a {
+      &:hover,
+      .active {
```
Comment (C-Lodder) [inline media/plg_installer_webinstaller/scss/client.scss]: Not checked the markup, but assume this should be `&.active`

File: media/plg_installer_webinstaller/scss/client.scss [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+}
+
+.ifw-loading-container {
+  top: 0;
+  left: 0;
+  z-index: 1000;
+  width: 100%;
+  height: 100%;
+  background: url("../../system/images/ajax-loader.gif") rgba(255,255,255,.7) 50% 10px no-repeat;
+  filter: alpha(opacity = 95);
```
Comment (C-Lodder) [inline media/plg_installer_webinstaller/scss/client.scss]: You can remove this line

File: media/plg_installer_webinstaller/scss/client.scss [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    .badge-#{$color} {
+      @include badge-variant($value);
+    }
+  }
+
+  .fader {
+    position: absolute;
+    top: 0;
+    left: 0;
+    float: left;
```
Comment (C-Lodder) [inline media/plg_installer_webinstaller/scss/client.scss]: You shouldn't need `float: left` if you're already absolutely positioning the element to the left.

File: media/plg_installer_webinstaller/scss/client.scss (hunk 1 above)
Comment (mbabker) [reply media/plg_installer_webinstaller/scss/client.scss]: Yep, fixed.

File: media/plg_installer_webinstaller/scss/client.scss (hunk 2 above)
Comment (mbabker) [reply media/plg_installer_webinstaller/scss/client.scss]: Done.

File: media/plg_installer_webinstaller/scss/client.scss (hunk 3 above)
Comment (mbabker) [reply media/plg_installer_webinstaller/scss/client.scss]: Probably not, but it was already there so I left it.  Either way, gone now.

**Review by C-Lodder — APPROVED** [review]

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 89e69649893d425bbd909abf4bdee16f5fd8d8a2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19428">issues.joomla.org/tracker/joomla-cms/19428</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 89e69649893d425bbd909abf4bdee16f5fd8d8a2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19428">issues.joomla.org/tracker/joomla-cms/19428</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Pull request merged**